src/api/code-challenge/content-types/code-challenge/lifecycles.js

```diff
@@ -1,43 +1,131 @@
+const pMap = require("p-map");
+const pick = require("lodash/pick");
+const { removeEmpty } = require("../code-challenge/general");
+const { runTestEvaluator } = require("../code-challenge/test-evaluator");
 const { handleInternalLabel } = require("../../../../utils/general");
+const { compareIds } = require("../code-challenge/utils");
 
-// runMetaTestsNeed more specific name later, maybe use "validateInternalTests"
-async function runMetaTests(eventTests, eventMetaTests) {
+const conductMetaTestsOnTests = async (
```

Comment:
yeah I think runMetaTests would be a much better name. Why did you get rid of it?

Reply:
I changed back to runMetaTests. I didn't think it was descriptive enough. I am getting the feeling the naming conventions are somewhat subjective. 

Is there some standard I can read on naming conventions? They exist in other fields like chemistry etc. Nomenclature?